staging/src/k8s.io/apiserver/pkg/server/healthz/healthz.go

```diff
@@ -58,13 +60,19 @@ func NamedCheck(name string, check func(r *http.Request) error) HealthzChecker {
 	return &healthzCheck{name, check}
 }
 
-// InstallHandler registers a handler for health checking on the path "/healthz" to mux.
+// InstallHandler registers handlers for health checking on the path
+// "/healthz" to mux. *All handlers* for mux must be specified in
+// exactly one call to InstallHandler. Calling InstallHandler more
+// than once for the same mux will result in a panic.
 func InstallHandler(mux mux, checks ...HealthzChecker) {
 	if len(checks) == 0 {
+		glog.Info("No default health checks specified. Installing the ping handler.")
 		checks = []HealthzChecker{PingHealthz}
 	}
+	glog.Infof("Installing %v healthz checkers", len(checks))
```

Comment:
We're already looping through the checks to install them. This runs once at startup and shouldn't overwhelm any logs. 

I don't think the cognitive overhead of writing a loop to accumulate all the names is worth it, but I can do that if you want.